requirements.txt

```diff
@@ -1,3 +1,4 @@
+requests
```

Comment:
Please move `requests` to `requirements-to-freeze.txt` and change this file's entry to `requests==2.18.4`.  It's a little more work to keep on the latest version of packages, but it ensures build stability (a travis build has failed at least once due to a minor version of boto3 changing).

Reply:
Oh, yeah, forgot about that!